server/index.js

```diff
@@ -19,6 +19,12 @@ app.get('/api/health-check', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products', (req, res, next) => {
+  db.query('select "productId", "name", "price", "image", "shortDescription" from "products"')
```

Comment:
You can also save your query to a const and pass it to the db.query, which would be useful when your queries start getting really large. Great work!!